conformance/setup.go

```diff
@@ -322,10 +322,7 @@ func init() {
 		runContentDiscoverySetup = false
 	}
 
-	skipEmptyLayerTest, _ = strconv.ParseBool(os.Getenv(envVarPushEmptyLayer))
-	if v, ok := os.LookupEnv(envVarDeleteManifestBeforeBlobs); ok {
-		deleteManifestBeforeBlobs, _ = strconv.ParseBool(v)
-	}
```

Comment:
are the changes related to `deleteManifestBeforeBlobs` intentional (this may have been my fault)

Reply:
Rebased now.